misaligned/MisalignedTwistedColorPairTest.java

```diff
@@ -0,0 +1,31 @@
+import java.util.List;
+
+public class MisalignedTwistedColorPairTest {
+
+    private static void testColorPairCombination(TwistedColorPair twistedColorPair) {
+
+        String expectedMajorColor = "White";
+        String expectedMinorColor = "Blue";
+        System.out.println(twistedColorPair.getMajorColor() + ": " + twistedColorPair.getMinorColor());
+        assert (expectedMajorColor.equalsIgnoreCase(twistedColorPair.getMajorColor())):" Invalid Major Color";
+        assert (expectedMinorColor.equalsIgnoreCase(twistedColorPair.getMinorColor())):"Invalid Minor Color";
+
+    }
+
+    public static void main(String[] args) {
+        final String[] majorColors = {"White", "Red", "Black", "Yellow", "Violet"};
+        final String[] minorColors = {"Blue", "Orange", "Green", "Brown", "Slate"};
+        TwistedColorPairWiring twistedColorPairWiring = new TwistedColorPairWiringImpl();
+        TwistedColorPairPrinter twistedColorPairPrinter = new TwistedColorPairPrinterImpl();
+        List<TwistedColorPair> twistedPairList = twistedColorPairWiring.generateTwistedColorPairCombination(majorColors,minorColors);
+        printTwistedPairColorCode(twistedColorPairPrinter,twistedPairList);
+        testColorPairCombination(twistedPairList.get(0));
```

Comment:
how do we verify what is printed as reference manual is aligned properly?
Test cases need to be added to verify the output. Please add suitable tests.

Reply:
Print Method has a return type of void as per clean code practice return type cannot be changed for test cases. It is only  possible  to check if method was called or not but that requires dependencies such as Mockito to check invocation of method or how many number of times it was called. So can you let me know what can be done here

Reply:
In such cases we try to cache the formatted strings in a list and verify as part of test asserts. We know that print function just prints what is provided. But we need to verify the input provided to print is indeed what is expected.

Reply:
Please find the latest commit part of rework